src/components/ui/sidebar.tsx

```diff
@@ -197,6 +202,7 @@ const Sidebar = React.forwardRef<
     if (isMobile) {
       return (
         <Sheet open={openMobile} onOpenChange={setOpenMobile} {...props}>
+          <SheetDescription className="sr-only">sidebar sheet</SheetDescription>
```

Comment:
End user doesn't necessarily know what a sheet is.
```suggestion
          <SheetDescription className="sr-only">Sidebar</SheetDescription>
```

Reply:
updated